src/enforcer.js

```diff
@@ -834,34 +987,42 @@ export class TrustedTypesEnforcer {
       console.warn(message, propertyName, context, typeToEnforce, value);
     }
 
+    // This is not available in release Firefox :(
```

Comment:
Can't reproduce. Firefox has them enabled since ~2 years ago.

Reply:
So we get rid of this feature test and just `this.windowObject.SecurityPolicyViolationEvent`?

Reply:
We still need it for, e.g. IE.

Reply:
Done.